CMakeLists.txt

```diff
@@ -241,7 +241,6 @@ set(
     ${MONERO_PROJECT_SRC}/wallet/message_store.cpp
     ${MONERO_PROJECT_SRC}/wallet/message_transporter.cpp
     ${MONERO_PROJECT_SRC}/wallet/node_rpc_proxy.cpp
-    ${MONERO_PROJECT_SRC}/wallet/wallet_rpc_payments.cpp
```

Comment:
This file is present in the release-v0.18 branch, which is what monero-ts is built on: https://github.com/monero-project/monero/blob/release-v0.18/src/wallet/wallet_rpc_payments.cpp